compiler/ir/tsl/tiled_strided_layout.py

```diff
@@ -170,17 +171,18 @@ def largest_common_contiguous_block(
             # check if contiguous block is found
             if next_stride is None:
                 return result or default_result
-            # return if dynamic
-            if next_stride[2].is_dynamic():
-                return result or default_result
             else:
```

Comment:
This is the main change of the PR, to let this continue if it encounters something dynamic.